backend/src/main/resources/liquibase/db-changelog-master.xml

```diff
@@ -5,10 +5,36 @@
 
     <include file="src/main/resources/liquibase/db-changelog-1.0.xml"/>
 
+    <changeSet author="anandJ" id="before-demo-table">
```

Comment:
these changes should be in db-changelog-2.0.xml  ? for POC to be closed to reality?

Reply:
it doesn't matter where the changeset tag is, but yes for the POC I can moved it to db-changelog-2.0.xml